specification/contosowidgetmanager-new2/resource-manager/readme.md

```diff
@@ -0,0 +1,48 @@
+# containerstorage
```

Comment:
@JeffreyRichter Should the readme.md file but under the service/rp folder or in the root of `resource-manager` folder?

Reply:
For RP & data-plane, there is one readme.md file per service in the ServiceName folder. This readme.md is required for docs & for SDK creation from swaggers (but not from TypeSpec).
The RP Namespace folder also needs a readme.md file required for RPaaS schema validation.